### PR TITLE
RIA-3275 fixing duplicate directions

### PR DESCRIPTION
**Pull request by burale**
### JIRA link (if applicable) ###

[RIA-3275](https://tools.hmcts.net/jira/browse/RIA-3275)

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AddAppealResponseHandler.java [hunk 1]
```diff
@@ -117,9 +118,13 @@ public boolean canHandle(
         asylumCase.write(RESPONDENT_DOCUMENTS, allRespondentDocuments);
 
         asylumCase.write(APPEAL_RESPONSE_AVAILABLE, YES);
-        asylumCase.write(REVIEW_RESPONSE_ACTION_AVAILABLE, YES);
+        asylumCase.write(REVIEW_RESPONSE_ACTION_AVAILABLE, NO);
         asylumCase.write(AMEND_RESPONSE_ACTION_AVAILABLE, YES);
 
+        asylumCase.write(REVIEW_HOME_OFFICE_RESPONSE_BY_LEGAL_REP, YES);
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AddAppealResponseHandler.java:124]: Should we consider to schedule automatic direction for timed event service? It is a part of `requestResponseReview`
@Srihari-Nakkala @burale 
Automatic direction will not happen unless `requestResponseeReview` is triggered, which that event excludes from the flow. What do you think?

Comment (makowalski) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AddAppealResponseHandler.java:124]: @burale did you clarify with Pav and Srihari about sending automatic direction

Comment (burale) [reply src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AddAppealResponseHandler.java:124]: It is clarified with Pav and automatic direction for timed event added.

**Comment by tarun-palisetty** [comment]
Created a new PR of the same https://github.com/hmcts/ia-case-api/pull/535 so closing this.

**Pull request closed without merging**